src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -229,100 +297,44 @@ export class CardItem extends React.Component<DevFileProps, {
                 id={`modal-${selectedVersion.name}`}>
                 <div className={this.props.cardItemStyle.helmCardHeader}>
                     <div className={this.props.cardItemStyle.devPageCardHeader}>
-                        <div className={this.props.cardItemStyle.devPageTitle}>
-                            <img
-                                data-testid='icon'
-                                src={this.state.selectedVersion.icon ? this.state.selectedVersion.icon : require('../../../../images/helm/helm.svg').default}
-                                alt={this.state.selectedVersion.icon + ' logo'}
-                                className={this.props.cardItemStyle.cardImage}
-                                style={{ margin: '0rem' }} />
-                            <div style={{ padding: '1rem', margin: '0rem' }}>
-                                <Typography variant='subtitle1'>
-                                    {
-                                        capitalizeFirstLetter(this.props.helmEntry.displayName)
-                                    }
-                                </Typography>
+                        <div style={{ display: 'flex', flexDirection: 'row', width: '100%', position: 'fixed' }}>
+                            <div className={this.props.cardItemStyle.devPageTitle} style={{ width: '70%', gap: '2rem' }}>
```

Comment:
Having width fixed to 70% is not useful if we need dynamic layout. For small screens, it will go out of modal view.

<img width="1344" alt="Screenshot 2023-05-21 at 10 51 26 PM" src="https://github.com/redhat-developer/vscode-openshift-tools/assets/3133356/5c400f94-45bd-4147-aa70-0d8b0a546f35">